docs/how_to/index.rst

```diff
@@ -0,0 +1,60 @@
+.. _how_to_guide:
+
+*************
+How-To Guides
+*************
+
+These guides provide examples of how to perform specific tasks with sunpy.
+These are recipes that do not provide much in depth explanation and assume you have some knowledge of what ``sunpy`` is and how it works.
+If you're starting fresh you might want to check out the :ref:`tutorial` first.
+
+.. toctree::
+   :maxdepth: 1
+
+   search_multiple_wavelengths.rst
+
+
+Quick Reference
+---------------
+
+The following table is meant as a quick reference.
+For more complete code examples, see the how-to guides above.
+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * - How do I...
+     - Solution
+   * - create a map from a FITS file
+     - :mod:`Map('file.fits') <sunpy.map>`
+   * - save a map to a FITS file
+     - :meth:`Map.save() <sunpy.map.GenericMap.save>`
```

Comment:
I thought we didn't need the `()` if we add `:meth:`?

Reply:
Then should we remove `:meth:`?

Reply:
Yeah I'll just remove them.